easybuild/easyconfigs/s/SDL2/SDL2-2.0.14-GCCcore-10.2.0.eb

```diff
@@ -22,7 +22,6 @@ toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
 
 source_urls = ['https://www.libsdl.org/release/']
 sources = [SOURCE_TAR_GZ]
```

Comment:
@fizwit please add the checksum.

Reply:
Actually, this easyconfig already exists. This PR is literally only removing the checksum.